server/lib/configuration.js

```diff
@@ -297,6 +297,11 @@ var conf = module.exports = convict({
       format: String,
       default: 'http://127.0.0.1:1114'
     },
+    api_timeout: {
+      doc: 'Timeout for talking to the Basket API server, in ms',
+      format: Number,
+      default: 5 * 1000 // 5 seconds
```

Comment:
Oh yeah, I forgot about that thing.  I mean, er, I was just following prevailing style in this file which already has numbers for durations in other places....